docker-images/crd-convert/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM strimzi/base:latest
```

Comment:
I guess we don't need the dockerfile though.

Reply:
Should I remove the whole docker-images/crd-convert dir then?

Reply:
@alesj